edison/src/main/java/edu/illinois/cs/cogcomp/edison/annotators/BrownClusterViewGenerator.java

```diff
@@ -107,6 +106,24 @@ private void loadClusters() throws Exception {
         if (gzip)
             stream = new GZIPInputStream(stream);
 
+        loadFromInputStream(stream);
+    }
+
+    public void loadFromDataStore() throws Exception {
+        Datastore dsNoCredentials = null;
+        try {
+            dsNoCredentials = new Datastore("http://smaug.cs.illinois.edu:8080");
+            System.out.println(file);
+            File f = dsNoCredentials.getFile("edu.cogcomp", file, 1.3);
+            InputStream is = new FileInputStream(f);
+            loadFromInputStream(is);
+        } catch (InvalidPortException | InvalidEndpointException e) {
+            e.printStackTrace();
+            System.out.println("The BrownCluster did not load. This is probably because out datastore server is off. Try again later or ping us. ");
```

Comment:
logger.error? (Also logger.error lets you provide the exception instance as an argument. So that appropriate information is shown in the logs.) https://www.slf4j.org/api/org/slf4j/Logger.html#error(java.lang.String, java.lang.Throwable)